pkg/synopsysctl/cmd_update.go

```diff
@@ -343,21 +344,26 @@ var updateBlackDuckCmd = &cobra.Command{
 				extraFiles = append(extraFiles, fmt.Sprintf("%s.yaml", size.(string)))
 			}
 
-			updateBlackDuckCobraHelper.SetArgs(instance.Config)
 			if err := util.UpdateWithHelm3(args[0], namespace, blackduckChartRepository, helmValuesMap, kubeConfigPath, extraFiles...); err != nil {
 				return err
 			}
+
+			err = blackduck.CRUDServiceOrRoute(restconfig, kubeClient, namespace, args[0], helmValuesMap["exposeui"], helmValuesMap["exposedServiceType"])
```

Comment:
We might also need to do this if it isOperatorBased -> What if a customer upgrades their operator and exposes the service at the same time.
(Thought: Maybe we should allow customers to set any other flags if they are upgrading to a new version)